ray-operator/controllers/common/pod.go

```diff
@@ -9,6 +9,7 @@ import (
 	rayiov1alpha1 "github.com/ray-project/kuberay/ray-operator/api/raycluster/v1alpha1"
 	"github.com/ray-project/kuberay/ray-operator/controllers/utils"
 
+	"github.com/ray-project/kuberay/ray-operator/controllers/common"
```

Comment:
My editor is complaining of a circular import here. Lmk if it's indeed a problem and how best to resolve. Probably by moving `GetHeadGroupServiceAccountName()` somewhere else?